### PR TITLE
Add an initial test for SSH.

### PR DESCRIPTION
**Pull request by brendandburns**
@quinton-hoole @cjcullen @krousey

### CODE REVIEW

File: pkg/util/ssh.go [hunk 1]
```diff
@@ -247,3 +267,91 @@ func GenerateKey(bits int) (*rsa.PrivateKey, *rsa.PublicKey, error) {
 	}
 	return private, &private.PublicKey, nil
 }
+
+type TestSSHServer struct {
```
Comment (cjcullen) [inline pkg/util/ssh.go]: Can this struct and the function below go into ssh_test.go? Is there a pkg/conditional compilation issue?

Comment (brendandburns) [inline pkg/util/ssh.go]: done.

**Comment by brendandburns** [comment]
comment addressed, ptal.

--brendan

**Comment by quinton-hoole** [comment]
Looks reasonably sane to me.  LGTM.

File: pkg/util/ssh_test.go [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+				newChannel.Reject(ssh.UnknownChannelType, fmt.Sprintf("unknown channel type: %s", newChannel.ChannelType()))
+				continue
+			}
+			channel, requests, err := newChannel.Accept()
+			if err != nil {
+				glog.Errorf("Failed to accept channel: %v", err)
+			}
+
+			for req := range requests {
+				glog.Infof("FOO: %v", req)
```
Comment (cjcullen) [inline pkg/util/ssh_test.go]: Maybe a better descriptor than "FOO" here.

**Comment by cjcullen** [comment]
LGTM, other than the 1 nit.

File: pkg/util/ssh_test.go (hunk 2 above)
Comment (brendandburns) [inline pkg/util/ssh_test.go]: done :P

**Comment by brendandburns** [comment]
comment addressed.  ready for submit.

**Pull request merged**